{"name": "Validar geocoder para 'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.'", "status": "failed", "statusDetails": {"message": "AssertionError: Error HTTP para NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.\nassert 400 == 200\n +  where 400 = <Response [400]>.status_code", "trace": "caso = {'archivo': 'direccion_total_cruces.json', 'direccion': 'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI...codificacion': 'Interseccion entre Calles', 'nombreCalle_1': 'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750)', ...}\n\n    @pytest.mark.parametrize(\"caso\", cargar_casos())\n    @allure.epic(\"Validaciones de geocoder\")\n    @allure.feature(\"Geocoder con cruces totales\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @allure.description(\"\"\"\n    Validación del endpoint **/geocoder** verificando que los atributos\n    `direccion`, `nombreCalle_1`, `nombreCalle_2`, `metodo_geocodificacion` y `tipoDireccion`\n    coincidan con lo definido en el archivo JSON.\n    \"\"\")\n    def test_datos_geocoder_con_atributos(caso):\n        direccion = caso[\"direccion\"]\n    \n        allure.dynamic.title(f\"Validar geocoder para '{direccion}'\")\n        allure.dynamic.parameter(\"Dirección\", direccion)\n        allure.dynamic.parameter(\"Archivo origen\", caso[\"archivo\"])\n    \n        with allure.step(\"Ejecutar request al endpoint\"):\n            response = requests.get(\n                ENDPOINTS[\"geocoder\"],\n                params={\"direccion\": direccion, \"v2\": \"true\"}\n            )\n            allure.attach(response.url, name=\"🔗 Request URL\", attachment_type=allure.attachment_type.TEXT)\n            allure.attach(response.text, name=\"📦 Response JSON\", attachment_type=allure.attachment_type.JSON)\n    \n        with allure.step(\"Verificar status code 200\"):\n>           assert response.status_code == 200, f\"Error HTTP para {direccion}\"\nE           AssertionError: Error HTTP para NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.\nE           assert 400 == 200\nE            +  where 400 = <Response [400]>.status_code\n\ntests_v2.0.3\\geocoder\\test_cruces_total.py:54: AssertionError"}, "description": "\nValidación del endpoint **/geocoder** verificando que los atributos\n`direccion`, `nombreCalle_1`, `nombreCalle_2`, `metodo_geocodificacion` y `tipoDireccion`\ncoincidan con lo definido en el archivo JSON.\n", "steps": [{"name": "Ejecutar request al endpoint", "status": "passed", "attachments": [{"name": "🔗 Request URL", "source": "aaee1bfa-44b4-4dd4-8985-aa6940e3afeb-attachment.txt", "type": "text/plain"}, {"name": "📦 Response JSON", "source": "c7ea6305-52a3-44da-b0f8-cb28553f8e19-attachment.json", "type": "application/json"}], "start": 1758723172148, "stop": 1758723172222}, {"name": "Verificar status code 200", "status": "failed", "statusDetails": {"message": "AssertionError: Error HTTP para NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.\nassert 400 == 200\n +  where 400 = <Response [400]>.status_code\n", "trace": "  File \"C:\\Users\\27960443689\\Desktop\\backup\\API_SERVICIO_GEO\\PY_TEST_APISG\\tests_v2.0.3\\geocoder\\test_cruces_total.py\", line 54, in test_datos_geocoder_con_atributos\n    assert response.status_code == 200, f\"Error HTTP para {direccion}\"\n"}, "start": 1758723172222, "stop": 1758723172223}], "parameters": [{"name": "caso", "value": "{'direccion': 'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.', 'nombreCalle_1': 'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750)', 'nombreCalle_2': 'RUGGIERI, SILVIO L.', 'metodo_geocodificacion': 'Interseccion entre Calles', 'tipoDireccion': 'calle_calle', 'archivo': 'direccion_total_cruces.json'}"}, {"name": "Dirección", "value": "'NORTE PJE. PARTICULAR (ALTURA SILVIO RUGGERI 2750) Y RUGGIERI, SILVIO L.'"}, {"name": "Archivo origen", "value": "'direccion_total_cruces.json'"}], "start": 1758723172148, "stop": 1758723172223, "uuid": "a4ae4e34-1fe1-446b-96d0-ad941870c1fc", "historyId": "e90d8cdcdc7a84f86e84f701b6d99424", "testCaseId": "619eca6dbb133aa78bebb5ab7e1ce151", "fullName": "tests_v2.0.3.geocoder.test_cruces_total#test_datos_geocoder_con_atributos", "labels": [{"name": "epic", "value": "Validaciones de geocoder"}, {"name": "feature", "value": "Geocoder con cruces totales"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests_v2.0.3.geocoder"}, {"name": "suite", "value": "test_cruces_total"}, {"name": "host", "value": "SECITD-P197C6K"}, {"name": "thread", "value": "9416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_v2.0.3.geocoder.test_cruces_total"}]}